
Train
Traceback (most recent call last):
  File "tools/train.py", line 176, in <module>
    main(args)
  File "tools/train.py", line 157, in main
    trainer = Trainer(args, cfg, device)
  File "/home/re6101029/CVPR2023comp/yolov6/yolov6/core/engine.py", line 148, in __init__
    self.train_loader, self.val_loader = self.get_data_loader(args, cfg, self.data_dict)
  File "/home/re6101029/CVPR2023comp/yolov6/yolov6/core/engine.py", line 580, in get_data_loader
    train_loader = create_dataloader(train_path, args.img_size, args.batch_size // args.world_size, grid_size,
  File "/home/re6101029/CVPR2023comp/yolov6/yolov6/data/data_load.py", line 42, in create_dataloader
    dataset = TrainValDataset(
  File "/home/re6101029/CVPR2023comp/yolov6/yolov6/data/datasets.py", line 71, in __init__
    self.img_paths, self.labels = self.get_imgs_labels(self.img_dir)
  File "/home/re6101029/CVPR2023comp/yolov6/yolov6/data/datasets.py", line 367, in get_imgs_labels
    assert 1 == 2
AssertionError